app/assets/stylesheets/table/table_panel/layer-views-panels/filters_panel.css.scss

```diff
@@ -105,7 +105,7 @@
       .jspCap { display: none; }
       .jspTrack { background: none; position: relative; }
       .jspDrag { background: rgba(#BBB, .5); @include border-radius(5px); position: relative; top: 0; left: 0; cursor: pointer; }
-      .jspArrow { background: none; text-indent: -20000px; display: block; cursor: pointer; }
+      .jspArrow { background: none; text-indent: -20000px; display: block; cursor: default; }
```

Comment:
Are we sure that this class is only used in the elements we want to prevent from showing pointer? It looks like it since this is `filters_panel.css.scss`, but it's never bad to check with a global search for this class name.

If you're 100% sure it's only used where we want the change to happen it's fine. Otherwise I would suggest you create a separate class (i.e `.jspArrowNoPointer`) and use it in all the elements we're interested in changing.

This is more "backwards compatible" fix. If not, we might remove the `pointer` from other places were we didn't intend to.